core/templates/pages/oppia-root/routing/access-validation-backend-api.service.ts

```diff
@@ -57,6 +57,9 @@ export class AccessValidationBackendApiService {
   BLOG_AUTHOR_PROFILE_PAGE_ACCESS_VALIDATOR = (
     '/access_validation_handler/can_access_blog_author_profile_page/<author_username>'); // eslint-disable-line max-len
 
+  COLLECTION_PLAYER_PAGE_ACCESS_VALIDATOR = (
```

Comment:
COLLECTION_PLAYER_PAGE_ACCESS_VALIDATOR_URL_TEMPLATE

Reply:
Updated it.